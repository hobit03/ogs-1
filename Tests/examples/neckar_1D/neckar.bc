GeoSys-BC: Boundary Conditions ------------------------------------------------
#BOUNDARY_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  POINT POINT1
 $DIS_TYPE
  CONSTANT 0

#BOUNDARY_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE 
  N(5)
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 4.84E-4 

#BOUNDARY_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE 
  S(6)
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 4.84E-4

#BOUNDARY_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE 
  O(0)
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 6e-4

#BOUNDARY_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE 
  K
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 1.5e-3

#BOUNDARY_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE 
  Toc
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 1e-8

#BOUNDARY_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE 
  pH
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 7.0

#BOUNDARY_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE 
  pe
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 4.0
#STOP

