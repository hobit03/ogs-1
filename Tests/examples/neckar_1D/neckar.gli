GeoSys-BC: Boundary Conditions ------------------------------------------------

#POINTS
0  0.0 0.0 0.0 $NAME POINT0
1  3 0.0 0.0 $NAME POINT1
2  1.5	0	0 $NAME POINT2


#POLYLINE
 $NAME 
  OUT_LINE
 $EPSILON
  1.0e-4
 $POINTS
  0
  1
#STOP
