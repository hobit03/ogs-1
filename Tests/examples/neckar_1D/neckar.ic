GeoSys-IC  Initial Conditions ------------------------------------------------
#INITIAL_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 1.0

#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE 
  N(5)
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1e-8

#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE 
  S(6)
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT  1e-8

#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE 
  O(0)
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 1e-8  //3e-4

#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE 
  K
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 1e-8  //3e-4
  
#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE 
  Toc
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 2

#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE 
  pH
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 7

#INITIAL_CONDITION
 $PCS_TYPE
  MASS_TRANSPORT
 $PRIMARY_VARIABLE 
  pe
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 4.0

#STOP

