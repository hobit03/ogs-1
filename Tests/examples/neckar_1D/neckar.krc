GeoSys Kinetic Reactions
#KINREACTIONDATA      
 $SOLVER_TYPE
	1                ; Solver type
 $RELATIVE_ERROR	
	1.e-6		 ; rel. Error
 $MIN_TIMESTEP	
	1.e-6		; min Timestep
 $INITIAL_TIMESTEP	
	1.e-4       ; init Timestep
 $BACTERIACAPACITY
  100.0
 $NO_REACTIONS
   POLYLINE PLY_BC_LEFT

#REACTION ; 
 $NAME
   oxidation_oxygen ;
 $TYPE
   USER_EXP;
 $EQUATION
  1 CH2O + 1 O2 =  1 CO2 + 1 H2O ; 
 $USER_EXPRESSION
  1.0 * 1.57E-9 * 10*mCH2O * mO2 / ( 2.94E-4 + mO2 )

#REACTION ; 
 $NAME
   oxidation_nitrate ;
 $TYPE
   USER_EXP;
 $EQUATION
  5 CH2O + 4 NO3n =  2 N2 + 4 HCO3n + 1 H2CO3 + 2 H2O ; 
 $USER_EXPRESSION
  1.0 * 1.57E-9 * mCH2O * 0.01 * mNO3n / ( 1.55E-4 + mNO3n ) / 5.0 

#REACTION ; 
 $NAME
   oxidation_sulfate ;
 $TYPE
   USER_EXP;
 $EQUATION
  2 CH2O + 1 SO4nn =  2 HCO3n + 1 H2S ; 
 $USER_EXPRESSION
  1.0 * 1.57E-9 * mCH2O * 6.4E-5 * mSO4nn / ( 1.0E-4 + mSO4nn ) / 2.0  
  
#STOP