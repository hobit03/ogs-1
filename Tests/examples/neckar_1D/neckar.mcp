#COMPONENT_PROPERTIES  ; comp1
 $NAME
  NO3n ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  1e-9             	; diffusion model type, diffusion constant

#COMPONENT_PROPERTIES  ; comp2
 $NAME
  SO4nn ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  1e-9             	; diffusion model type, diffusion constant

#COMPONENT_PROPERTIES  ; comp3
 $NAME
  O2 ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  1e-9             	; diffusion model type, diffusion constant

#COMPONENT_PROPERTIES  ; comp4
 $NAME
  CH2O ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  1e-9             	; diffusion model type, diffusion constant
  
#COMPONENT_PROPERTIES  ; comp5
 $NAME
  CO2 ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  1e-9             	; diffusion model type, diffusion constant
  
#COMPONENT_PROPERTIES  ; comp6
 $NAME
  H2O ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  1e-9             	; diffusion model type, diffusion constant
  
#COMPONENT_PROPERTIES  ; comp7
 $NAME
  N2 ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  1e-9             	; diffusion model type, diffusion constant
  
#COMPONENT_PROPERTIES  ; comp8
 $NAME
  HCO3n ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  1e-9             	; diffusion model type, diffusion constant
  
#COMPONENT_PROPERTIES  ; comp9
 $NAME
  H2CO3 ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  1e-9             	; diffusion model type, diffusion constant
  
#COMPONENT_PROPERTIES  ; comp10
 $NAME
  H2S ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $DIFFUSION
  1  1e-9             	; diffusion model type, diffusion constant
  
#STOP
