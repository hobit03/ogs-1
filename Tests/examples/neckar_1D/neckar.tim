GeoSys-TIM: Timestepping ------------------------------------------------
#TIME_STEPPING
 $PCS_TYPE
  GROUNDWATER_FLOW
 $TIME_STEPS
  100  158730
 $TIME_END
  15873000
 $TIME_START
  0.0
#TIME_STEPPING
 $PCS_TYPE
  REACT_TRANS_OPS
 $TIME_STEPS
  100  158730
 $TIME_END
  15873000
 $TIME_START
  0.0
#STOP

  50  158730
 $TIME_END
  158730200
 $TIME_START
  0.0

