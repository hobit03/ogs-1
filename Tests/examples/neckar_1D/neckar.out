GeoSys-OUT: Output ------------------------------------------------
#OUTPUT // profile
 $NOD_VALUES
  HEAD
  CH2O
  O2
  CO2
  H2O
  NO3-
  N2
  HCO3-
  H2CO3
  SO42-
  H2S
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 1
#STOP




  STEPS 5
  
#OUTPUT // profile
 $ELE_VALUES
  VELOCITY1_X
  VELOCITY1_Y
  VELOCITY1_Z
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  VTK
 $TIM_TYPE
   0.0
   100.0
   1000.0
   10000.0
   21000.0
